Makefile

```diff
@@ -15,17 +15,17 @@ cli: core
 	CARGO_PROFILE_RELEASE_LTO=off cargo build --package=javy-cli --release
 
 core:
-	cargo build --package=javy-core --release --target=wasm32-wasi --features=$(CORE_FEATURES)
+	cargo build --package=javy-core --release --target=wasm32-wasip1 --features=$(CORE_FEATURES)
 
 docs:
 	cargo doc --package=javy-cli --open
-	cargo doc --package=javy-core --open --target=wasm32-wasi
+	cargo doc --package=javy-core --open --target=wasm32-wasip1
 
 test-javy:
-	CARGO_TARGET_WASM32_WASI_RUNNER="wasmtime --dir=."  cargo wasi test --package=javy --features json,messagepack -- --nocapture
+	CARGO_TARGET_WASM32_WASIP1_RUNNER="wasmtime --dir=." cargo test --package=javy --target=wasm32-wasip1 --features json,messagepack -- --nocapture
 
 test-core:
-	cargo wasi test --package=javy-core -- --nocapture
+	CARGO_TARGET_WASM32_WASIP1_RUNNER="wasmtime" cargo wasi test --package=javy-core -- --nocapture
```

Comment:
Should this here be `cargo test` instead?

Reply:
Yes 🤦. I had originally updated this target to not use cargo-wasi but then deleted the target because we don't have any tests in `javy-core`. Then I decided to bring it back in case we ever add any test cases in the future. But when I restored it, I restored it to the original version and to the version where I had removed cargo-wasi.